### DISCUSSION TITLE
Overwrite the header values

### DISCUSSION THREAD

**Discussion by LindoZondo** [apideck-libraries/portman#611] [Q&A] [answered]
Hi Everyone 
I wish to find out how to overwrite the header values for a specific endpoint.

Thank you

**Comment by thim81** [comment, marked as answer]
hi @LindoZondo 

Best starting point is the [overwriteRequestHeaders docs](https://github.com/apideck-libraries/portman/tree/main#overwrites-options) and the [overwrite examples](https://github.com/apideck-libraries/portman/tree/main/examples/testsuite-overwrites)

Using the example of the overwriteRequestHeaders Portman directive, you would overwrite the request header for the specific `leadsUpdate` endpoint, where the `key` would point to the Header you wish to overwrite and `value` the new value you want to set:

```
{
  "overwrites": [
    {
      "openApiOperationId": "leadsUpdate",
      "overwriteRequestHeaders": [
        {
          "key": "x-apideck-consumer-id",
          "value": "portman-id-{{$randomInt}}",
          "overwrite": true
        }
      ]
    }
  ]
}
```

**Reply by LindoZondo** [reply]
Hey @thim81 Thanks I am going to give it a go, will let you know if it works.

**Reply by thim81** [reply]
@LindoZondo did it work for you?

**Reply by LindoZondo** [reply]
Hey @thim81 
It worked man, thank you so much.
